Azure/sonic-buildimage
# [platform mellanox] Substitute '-' symbol to '.' in Mellanox SDK vers…

…ion to prevent problems with finding deb package

Signed-off-by: Andriy Yurkiv <ayurkiv@nvidia.com>



**- Why I did it**
In case of non-GA SDK version there is '-' symbol in Mellanox SDK version name. (For example: 4.4.1306-006) 
In appropriate .deb packet there is '.' instead of '-'. Because of this there was problem while building SDK 
**- How I did it**
I have replaced  '-' symbol with '.' for MLNX_SDK_DEB_VERSION
**- How to verify it**
1) build SDK (non-GA version with suffix '-00X') without fix. Building will fail because it is impossible to find   appropriate package 
2) build SDK (non-GA version with suffix '-00X') with fix. Everything should be OK

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
